lib/phlex/view/callbacks.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Phlex::View::Callbacks
+	def template(&block)
+		respond_to?(:before_template) && before_template
+
+		super(&block)
+
+		respond_to?(:after_template) && after_template
```

Comment:
Nit: I think these might read better written like this.

```suggestion
		after_template if respond_to?(:after_template)
```